src/ui/components/Passport/Passport.tsx

```diff
@@ -58,10 +60,10 @@ const Passport = (props: PropsFromRedux) => {
   const [videoHeight, setVideoHeight] = useState<number | null>(null);
 
   useEffect(() => {
```

Comment:
:question: can you clarify what the intended logic flow is with this `useEffect` ?

Reply:
Sure – I want to show this banner only a single time. So if you go to the page, you'll see the banner, but then if you switch tabs and return (or hit reload) the tab should disappear.

So this code is meant to do that.